docs/faq.rst

```diff
@@ -20,3 +20,19 @@ A: No. ``@most/core`` is focused on reactive event programming rather than Array
 **Q: I want to stream chunked data with flow control (also known as "back pressure"). Should I use ``@most/core`` for that?**
 
 A: No. ``@most/core`` is focused on reactive event programming rather than chunked/block data streaming :ref:`Read more here <when should you use it>`.
+
+**Q: Are ``@most/core`` event streams the same as Node streams?**
+
+A: No. ``@most/core`` and Node streams are both based on the general concept of [stream processing](https://en.wikipedia.org/wiki/Stream_processing).  However, they differ in their specific goals, architectures, and APIs.
+
+``@most/core`` :ref:`Event Streams`:
+
+* Deal with discrete events as they happen, such as mouse clicks, where reactivity (timeliness) is a significant factor;
```

Comment:
Remove the initial capital letter of each bullet point. Please refer to my comment [here](https://github.com/mostjs/core/pull/261#pullrequestreview-151145781) on these types of vertical lists.